doc/source/ray-air/examples/torch_incremental_learning.ipynb

```diff
@@ -246,7 +246,7 @@
     "This will allow us to apply built-in preprocessors to the Ray Dataset and allow Ray Datasets to be used with Ray AIR Predictors.\n",
     " <!-- and also means that any transformations done to the images can be done in a zero-copy fashion. -->\n",
     "\n",
-    "For this example, since we are just working with MNIST dataset, which is small, we use the [`SimpleTorchDataSource`](https://docs.ray.io/en/master/data/package-ref.html?highlight=SimpleTorchDatasource#ray.data.datasource.SimpleTorchDatasource) which just loads the full MNIST dataset into memory.\n",
+    "For this example, since we are just working with MNIST dataset, which is small, we use the [`SimpleTorchDataSource`](https://docs.ray.io/en/master/data/api/input_output.html?highlight=SimpleTorchDatasource#ray.data.datasource.SimpleTorchDatasource) which just loads the full MNIST dataset into memory.\n",
```

Comment:
we should update this to be a proper internal ref

Reply:
🤯  updated!